docs/overview.md

```diff
@@ -74,9 +74,34 @@ Comparison                      | `>`, `<`, `>=`, `<=`
 Boolean operations              | `!`, `&&`, <code>&#124;&#124;</code>
 Reference equality              | `===`, `!==`
 Structural equality             | `==`, `!=`
+
+## Implicit Return
+
+There is no `return` keyword in Reason. The last expression in a block or
```

Comment:
Did we introduce "expression blocks" at this point in the docs yet? If not, I might include it before this.

Reply:
Removed expression blocks from the example.